2_process/src/munge_group_files.R

```diff
@@ -35,7 +35,7 @@ unzip_and_merge_files <- function(lake_ids, irradiance_zipfile, clarity_zipfile,
     
     return(tibble(filename = site_id_fn, hash = tools::md5sum(site_id_fn)))
   }) %>% 
-    purrr::reduce(bind_rows)
+    purrr::reduce(bind_rows) %>% filter(!filename == "") #remove empty
```

Comment:
Should we be concerned if any are empty? Put some message in to notify the person running the pipeline?

Reply:
Many of the lakes won't have PGDL predictions for temperature since PGDL requires the lake to have >= 10 profiles of temperature data.